test/functional/http/post/3.votes.js

```diff
@@ -0,0 +1,458 @@
+'use strict';
+
+var node = require('../../../node');
+var shared = require('../../shared');
+var constants = require('../../../../helpers/constants');
+
+var sendTransactionPromise = require('../../../common/apiHelpers').sendTransactionPromise;
+var creditAccountPromise = require('../../../common/apiHelpers').creditAccountPromise;
+var registerDelegatePromise = require('../../../common/apiHelpers').registerDelegatePromise;
+var waitForBlocksPromise = node.Promise.promisify(node.waitForBlocks);
+
+describe('POST /api/transactions (type 3) votes', function () {
+
+	var transaction;
+	var badTransactions = [];
+	var goodTransactions = [];
+	var badTransactionsEnforcement = [];
+	var goodTransactionsEnforcement = [];
+
+	var delegateAccount = node.randomAccount();
+	var accountNoFunds = node.randomAccount();
+	var accountMinimalFunds = node.randomAccount();
+	var accountDuplicates = node.randomAccount();
+
+	/*
+	 Creating two scenarios with two isolated set of accounts
+	
+	 - Firts scenario --> MaxVotesPerTransaction
+	   We register the exact amount of delegates the network's allows to be voted per transaction. 
+	   A new fresh account is creadited to perform the voting tests to those delegates.
+
+	 - Second scenario --> MaxVotesPerAccount
+	   In this case, we create as many delegates as the network allows to be active. This parameter
+       is directly related to the maximum number of allowed votes per account. Another independent 
+	   account is credited and performs tests uniquely to delegates from this scenario.
+	*/
```

Comment:
Fixed typos:
Creating two scenarios with two isolated set of accounts

- First scenario --> MaxVotesPerTransaction
We register the exact amount of delegates the networks allows to be voted per transaction.
A new fresh account is credited to perform the voting tests to those delegates.

- Second scenario --> MaxVotesPerAccount
In this case, we create as many delegates as the network allows to be active. This parameter
is directly related to the maximum number of allowed votes per account. Another independent
account is credited and performs tests uniquely to delegates from this scenario.